src/app/shared/services/map.service.ts

```diff
@@ -305,8 +314,14 @@ export class MapService {
     // Query Fire Perimeters
     public trace(geojson: any) {
         const httpOptions = { headers: new HttpHeaders({ 'Content-Type': 'application/json' }) };
-        return this._http.post<any>('https://firehydrology.streamstats.usgs.gov/trace', geojson, httpOptions);
+        return this._http.post<any>('https://firehydrology.streamstats.usgs.gov/trace', geojson, httpOptions)
+        .pipe(catchError((err: any) => {
+            this.createMessage("Error getting fire perimeters.", 'error');
```

Comment:
This error will show up if the fire perimeters are working correctly but the trace service has failed, so maybe a more accurate message is needed. "Error tracing fire perimeters"?